tests/integration/imagerepo_and_policy_test.go

```diff
@@ -25,18 +25,20 @@ import (
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 
+	"github.com/fluxcd/pkg/apis/meta"
+
 	imagev1 "github.com/fluxcd/image-reflector-controller/api/v1beta2"
 )
 
-func TestImageRepositoryScan(t *testing.T) {
+func TestImageRepositoryScanAndPolicyDigest(t *testing.T) {
 	for name, repo := range testRepos {
 		t.Run(name, func(t *testing.T) {
-			testImageRepositoryScan(t, repo)
+			testImageRepositoryScanAndPolicyDigest(t, repo)
 		})
 	}
 }
 
-func testImageRepositoryScan(t *testing.T, repoURL string) {
+func testImageRepositoryScanAndPolicyDigest(t *testing.T, repoURL string) {
```

Comment:
Let's run this for AWS/ECR also.

Reply:
Passing: https://github.com/fluxcd/image-reflector-controller/actions/runs/15001013945/job/42147679887